android/src/com/frostwire/android/gui/fragments/TransfersFragment.java

```diff
@@ -119,21 +118,21 @@ public void onActivityCreated(Bundle savedInstanceState) {
     }
 
     @Override
-    public void onViewCreated(View view, Bundle savedInstanceState) {
-        super.onViewCreated(view, savedInstanceState);
-        subscription = TimerService.subscribe(this, 2);
-    }
-
-    @Override
     public void onResume() {
         super.onResume();
         initStorageRelatedRichNotifications(getView());
+    }
+
+    @Override
+    public void onStart() {
+        super.onStart();
+        subscription = TimerService.subscribe(this, UI_UPDATE_INTERVAL_IN_SECS);
         onTime();
     }
 
     @Override
-    public void onDestroyView() {
-        super.onDestroyView();
+    public void onStop() {
+        super.onStop();
         subscription.unsubscribe();
```

Comment:
@aldenml have we checked if it actually unsubscribes when the fragment is invisible (I assume this un-subscription was here before), from the documentation I understand it only stops because the activity stopped.

>onStop() fragment is no longer visible to the user either because its activity is being stopped or a fragment operation is modifying it in the activity.